homeassistant/components/media_player/sonos.py

```diff
@@ -608,6 +613,13 @@ def update(self):
             else:
                 playlist_size = int(playlist_size)
 
+            if playlist_size is None or playlist_size == 0:
```

Comment:
Can you do something similar to the `soco_error` decorator? https://github.com/Juggels/home-assistant/blob/db8414048ee85cb63ac388b3f8a4b571a2f9074f/homeassistant/components/media_player/sonos.py#L248-L259